src/Elan.Web/Controllers/FriendsController.cs

```diff
@@ -14,21 +17,16 @@ namespace Elan.Web.Controllers
     public class FriendsController : ElanBaseController
     {
         private readonly IFriendsService _friendsService;
+        private readonly IFriendsInvitationService _friendsInvitationService;
         private readonly IUserService _userService;
+        private readonly INotificationService _notificationService;
 
-        public FriendsController(IFriendsService friendsService, IUserService userService)
+        public FriendsController(IFriendsService friendsService, IUserService userService, IFriendsInvitationService friendsInvitationService, INotificationService notificationService)
```

Comment:
Formatting: when class has many dependencies in its constructor we should shorten lines using vertical spacing before each dependency.